app/views/events/index.html.erb

```diff
@@ -38,32 +39,35 @@
         <div class="hidden-md-up">
           <%=  Date.commercial(@date.year, week_num, day_num).strftime("%A") %>
         </div>
-      <div class="col-md-3 calendar-cell">
-        <span class="calendar-date">
-        <%=  Date.commercial(@date.year, week_num, day_num).day %>
-        </span>
+        <div class="col-md-3 calendar-cell-hover">
+          <span class="calendar-date">
+            <%=  Date.commercial(@date.year, week_num, day_num).day %>
+          </span>
         </br>
         <% if @events[week_num][day_num].size > 0 %>
```

Comment:
Is it possible to remove all this logic (lines 47 to 69) from the template and maybe add some tests to make sure the problem was fixed?

Reply:
I have moved the logic to the helper, written tests for the logic.  

I was wondering if you would suggest writing acceptance tests or if the tests added for the logic would suffice?